src/community/geofence-server/src/main/resources/applicationContext.xml

```diff
@@ -56,7 +56,8 @@
         <property name="locations">
             <list>
                 <value>classpath*:geofence-datasource-ovr.properties</value>
-                <value>${GEOSERVER_DATA_DIR}/geofence/geofence-datasource-ovr.properties</value> <!-- optional -->
+                <!-- This location is relative to the datadir -->
+                <value>file:geofence/geofence-datasource-ovr.properties</value> <!-- optional -->
```

Comment:
@NielsCharlier we use file: URIs like this one in several places in the code base (including app-schema) but they are never a valid file: URI because any file: URI with a scheme (i.e. "file:") must be absolute. This construct will always elicit a grumble from me. Perhaps @jodygarnett can comment.